doc/classes/Shader.xml

```diff
@@ -16,6 +17,7 @@
 			<argument index="0" name="param" type="String">
 			</argument>
 			<description>
+				Returns the texture that is set as default for specified parameter. [b]Note:[/b] [code]param[/code] must match the name of the uniform in the code exactly.
```

Comment:
for *the* specified parameter (I think? I'm always asking people to add missing "the"s in their PRs, might be a Frenchism :P).